services/nexus/nexus.tester.js

```diff
@@ -1,71 +1,82 @@
 'use strict'
 
 const Joi = require('joi')
-const { invalidJSON } = require('../response-fixtures')
+const sinon = require('sinon')
 
+const { colorScheme } = require('../test-helpers')
+const {
+  isVPlusDottedVersionNClausesWithOptionalSuffix: isVersion,
+} = require('../test-validators')
 const t = (module.exports = require('../create-service-tester')())
+const serverSecrets = require('../../lib/server-secrets')
 
-t.create('search release version')
+const user = 'admin'
+const pass = 'password'
+
+function mockNexusCreds() {
+  serverSecrets['nexus_user'] = undefined
+  serverSecrets['nexus_pass'] = undefined
+  sinon.stub(serverSecrets, 'nexus_user').value(user)
+  sinon.stub(serverSecrets, 'nexus_pass').value(pass)
+}
+
+t.create('live: search release version valid artifact')
   .get('/r/https/repository.jboss.org/nexus/jboss/jboss-client.json')
   .expectJSONTypes(
     Joi.object().keys({
       name: 'nexus',
-      value: Joi.string().regex(/^v4(\.\d+)+$/),
+      value: isVersion,
     })
   )
 
-t.create('search release version of an inexistent artifact')
+t.create('live: search release version of an inexistent artifact')
```

Comment:
This comes up a lot. Maybe we should add something to the test runner which includes `[mock]` next to all the ones which use mocks (or vice versa).